### PR TITLE
Add minimalist `subscribeCurrentUserUpdates` script

### PR DESCRIPTION
**Pull request by antoine-malliarakis**
We were frequently being reported issues with issues with JWT based authentication when trying to subscribe for current user data.

This is not an attempt to fix the issue itself but the reference script we used to validate the behaviour and the fix

### CODE REVIEW

**Comment by redox** [comment]
@antoine-malliarakis let's also link this file from the README patching/creating a small section. Very useful

**Comment by antoine-malliarakis** [comment]
> @antoine-malliarakis let's also link this file from the README patching/creating a small section. Very useful

Indeed. Added.

File: README.md [hunk 1]
```diff
@@ -960,6 +960,61 @@ cable.subscribe("aCardWasUpdated { slug }", {
...
+    }
+    const { slug } = currentUserWasUpdated;
+    console.log('current user was updated', slug);
+    process.exit(0);
+  }
+});
+
+```
+
+This example can be found in [examples/subcribeCurrentUserUpdates.js](./examples/subscribeCurrentUserUpdates.js) which can be run with the following environment parameters:
```
Comment (utay) [inline README.md:1013]: I feel like linking the script here is enough and we shouldn't add code to the README as it's already big, wdyt?

**Review by utay — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (antoine-malliarakis) [reply README.md:1013]: Mmmmmm the idea of this read me is that people could get enticed to copy paste it and make it clean. If we only leave the reference file they could be tempted to keep the environment variable logic (which may not be the best idea).

But I give you that it's debatable IMHO. WDYT ?

**Pull request merged**